corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/form.less

```diff
@@ -31,22 +31,15 @@
     background-color: lighten(@cc-brand-hi, 10);
     .border-bottom-radius(0);
 
-    .nonanchored-submit .btn {
+    .submit .btn {
       font-size: 20px;
       padding: 13px 24px;
       .transition(all .5s);
     }
-  }
 
-  .anchored-submit {
-    background-color: @call-to-action-low;
-    width: 100vw;
-    padding-top: 5px;
-    padding-bottom: 5px;
-    position: fixed;
-    bottom: 0px;
-    left: 0;
-    z-index: @zindex-formplayer-anchored-submit;
```

Comment:
wherever `zindex-formplayer-anchored-submit` should be able to be deleted now too